docs/source/examples.rst

```diff
@@ -53,7 +53,11 @@ can be used, called ``MoabCluster``:
 SGE Deployments
 ---------------
 
-On systems which use SGE as the scheduler, ``SGECluster`` can be used:
+On systems which use SGE as the scheduler, ``SGECluster`` can be used.
+
+SGE systems have a lot of flexibility in how they are configured, hence it is not possible to use the ``memory`` keyword argument to automatically specify the amount of RAM used. Instead, you must request for resources according to how your system is configured.
```

Comment:
"the amount of RAM requested" ?
"Instead, you should specify the resources (...), using the resource_spec kwarg."

Reply:
@guillaumeeb:

> "the amount of RAM requested" ?

Yes! Thank you for the catch.

> "Instead, you should specify the resources (...), using the resource_spec kwarg."

Yes again! I will change both.

Reply:
Add the link to the distributed doc about dask memory management: http://distributed.dask.org/en/latest/worker.html#memory-management